workitem/workitem_repository.go

```diff
@@ -588,6 +588,9 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, spaceID uuid.UUID,
 		if err != nil {
 			return nil, errors.NewBadParameterError(fieldName, fieldValue)
 		}
+		if (fieldName == SystemAssignees || fieldName == SystemLabels) && fieldValue == nil {
+			wi.Fields[fieldName] = make([]interface{}, 0)
+		}
```

Comment:
Can we add a test for creation as well?
Means, you can add assertions into existing test cases for WI create.

Reply:
Fixed in bea6a4f6096c238347e1136e4c6ff22172da494e